src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -62,7 +73,30 @@ class EntityClient (requestContext: RequestContext) extends Actor {
         requestContext.failWith(
           new RequestProcessingException(StatusCodes.InternalServerError, error.getMessage))
     }
+  }
 
+  def createEntities(wsNamespace: String, wsName: String, entities: Seq[Entity]) = {
+    val results = entities.map(entity => createEntityOrReportError(wsNamespace, wsName, entity))
+    // have to manually construct an HttpResponse here because we're aggregating the results of multiple calls to Rawls
+    val response = HttpResponse(OK, HttpClient.createJsonHttpEntity(results.toJson.compactPrint))
```

Comment:
We will be implementing the full featured upload next sprint (DSDEEPB-4), this is just a first pass effort we could finish in this sprint.  For now, we don't need to worry about clean up, or whether or not all succeeded.  I think if we just do OK upon successful call and return success/failure for each entity.  In the next sprint we will be adding a batch create/update method that will work on a single transaction and rollback any changes if there are failures.